apps/console/src/features/applications/components/settings/info.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { LoadableComponentInterface, TestableComponentInterface } from "@wso2is/core/models";
+import { ContentLoader, EmphasizedSegment, Heading } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { Divider, Grid } from "semantic-ui-react";
+import { applicationConfig } from "../../../../extensions";
+import { AppState } from "../../../core";
+import {
+    InboundProtocolListItemInterface,
+    OIDCApplicationConfigurationInterface,
+    SAMLApplicationConfigurationInterface
+} from "../../models";
+import { OIDCConfigurations, SAMLConfigurations } from "../help-panel";
+
+/**
+ * Proptypes for the server endpoints details component.
+ */
+interface InfoPropsInterface extends LoadableComponentInterface, TestableComponentInterface {
+    inboundProtocols: InboundProtocolListItemInterface[];
+    isSAMLConfigLoading: boolean;
+    isOIDCConfigLoading: boolean;
+}
+export const Info: FunctionComponent<InfoPropsInterface> = (
```

Comment:
Lets add a component doc comment here.
```suggestion

export const Info: FunctionComponent<InfoPropsInterface> = (
```

Reply:
Added